content/tagging/using_tags.md

```diff
@@ -12,105 +12,271 @@ further_reading:
 
-To group using tags, enter the key part of the tag in the `avg by:` textbox.
+{{< img src="tagging/using_tags/dashboardtags.png" alt="Tags in Dashboards avg by textbox" responsive="true" style="width:80%;">}}
 
-For instance, if you have a timeseries graph showing a metric tagged by the reporting hosts roles -`role:database`, `role:frontend`, or `role:loadbalancer`- enter role in the **avg_by** textbox.
-This causes the graph to show just one line for each tag value - `database`, `frontend`, and `loadbalancer`. Each line represents the average metric value across all hosts that share that role.
+You can also use tags to overlay events on the dashboard. This works the same way as in the [Events Stream][2].
+Enter `tags:` followed by the tag. The matching events are overlaid as vertical bars on the graph. The example below uses `tags:service:coffee-house`.
 
-{{< img src="tagging/dashboardtags.png" alt="Tags in Dashboards avgby textbox" responsive="true" style="width:70%;">}}
+{{< img src="tagging/using_tags/dashboardeventtags.png" alt="Event Overlays in Dashboards" responsive="true" style="width:80%;">}}
 
-You can also use tags to overlay events on the dashboard. This works in exactly the same way as in the [Events List][2].
-Enter `tags:` followed by the tag and you see the corresponding events overlaid as vertical bars on each graph.
+Use [template variables][5] to save time switching the **from** tag on graphs in your dashboard. In the example below, `service` is used to represent the `service` tag group. To use the template variable, replace any tag listed in the **from** textbox with `$service`.
 
-## Using tags in the Infrastructure List and the Host Map
+{{< img src="tagging/using_tags/dashboardtemplatevariables.png" alt="Dashboard Template Variables" responsive="true" style="width:80%;">}}
 
-To filter the list of hosts in the [Infrastructure list][4], enter a tag in the filter textbox at the top of the page. You can also group the hosts by entering the key portion of the tag in the group by textbox. So if you enter role in the group box, you see each role as a group heading followed by the hosts with that tag.  
+[2]: /graphing/event_stream/
+[4]: /graphing/dashboards/
+[5]: /graphing/dashboards/template_variables/
+
+{{% /tab %}}
+{{% tab "Examples" %}}
+
+Here is an example of tags using the time-series chart editor. For the first screenshot, no tags have been applied, and we're observing average CPU across all hosts:
+
+{{< img src="tagging/using_tags/Tags_1.png" alt="Tags_1" responsive="true" style="width:75%;">}}
+
+Next, we've updated the editor to include a tag (`region:eastus`) in the **from** textbox that enables Datadog to look at CPU across the US East Region. We've used region as an example, but you could use any arbitrary tag, including application, service, environment, etc.
```

Comment:
`Next, we've updated the editor to include a tag` => `Next, the editor is updated to in clude a tag..`

`We've used region as an example, but you could use any arbitrary tag, including application, service, environment, etc.` => ```The `region` tag is used as an example here, but you could use any arbitrary tag sent to your Datadog Platform, including `application`, `service`, `environment`, etc...```